packages/isar/lib/src/native/binary_writer.dart

```diff
@@ -219,7 +219,9 @@ class BinaryWriter {
   }
 
   void validate() {
-    assert(_dynamicOffset == _buffer.length,
-        'Invalid write buffer size. $_dynamicOffset != ${_buffer.length}');
+    assert(
+      _dynamicOffset == _buffer.length,
+      'Invalid write buffer size. $_dynamicOffset != ${_buffer.length}',
+    );
```

Comment:
In addition, I fixed lint error.
Please let me exclude Codecov warnings from this PR scope.